v5/pivotal/stories.go

```diff
@@ -211,17 +213,66 @@ func (service *StoryService) List(projectID int, filter string) ([]*Story, error
 	return stories, nil
 }
 
-func newStoriesRequestFunc(client *Client, projectID int, filter string) func() *http.Request {
+func newStoriesRequestFunc(client *Client, projectID int, filter string, fields []string) func() *http.Request {
 	return func() *http.Request {
 		u := fmt.Sprintf("projects/%v/stories", projectID)
 		if filter != "" {
 			u += "?filter=" + url.QueryEscape(filter)
+			if len(fields) == 0 {
+				fields = DefaultFields
+			}
+			u += "&fields=" + url.QueryEscape(fieldsToQuery(fields))
```

Comment:
Some info about this,
If we don't explicitly request for some fields like `requested_by`, it doesn't return it in the response. However, the use of this field causes only the mentioned fields to appear. So, we have to fill this parameter with all the default fields again.